lib/src/components/sidebar/Sidebar.js

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Layout } from 'lib';
+import { SidebarHeader } from './SidebarHeader';
+import { SidebarBody } from './SidebarBody';
+import { SidebarSection } from './SidebarSection';
+import { SidebarHeading } from './SidebarHeading';
+import { SidebarSubHeading } from './SidebarSubHeading';
+
+export function Sidebar({ children, className = '', ...rest }) {
+  return (
+    <Layout className={`sidebar ${className}`} {...rest}>
```

Comment:
All `Sidebars` adopt the `Layout` to inherit the overflow scroll behaviour that we abstracted into `Layout.Body`.

The header & footer is then fixed as expected.

In our implementation we failed to do this (cause GUI didn't provide this out the box) and some sidebars don't overflow scroll 🤦

Reply:
This is no longer the case.